package-res/ccc/core/base/chart/chart.axes.js

```diff
@@ -445,6 +492,53 @@ pvc.BaseChart
             }
         }
 
+        // CDF603 C
+        // If min is null, but FixedMax and FixedLength were defined 
+        // the minimum can be set using both
+        if(min == null && width != null && 
+           max != null && axis.option.isDefined('FixedMax') ) {
+            min = max - width;
+            minLocked = (min != null);
+            if(minLocked) {
+                // this can and will change the length if 
+                // width > max
+                if(min < 0 && axis.scaleUsesAbs()) min = -min;
+            }
+        } 
+
+        // CDF603
+        // If min and max are null, but FixedLength was defined 
+        // the maximum and minimum are set according to the specified 
+        // or default alignment
+        if(min == null && max==null && width != null) {
```

Comment:
Incorrect spacing of operators... `max==null`, `axis.option('DomainAlign')=='max'`, etc.